Transfer/Sources/TransferAmountCell.swift

```diff
@@ -23,18 +23,24 @@ import Common
 final class TransferAmountCell: UITableViewCell {
     static let reuseIdentifier = "transferAmountCellIdentifier"
     typealias EnteredAmountHandler = (_ value: String) -> Void
+    let decimalSymbol = Character(NumberFormatter().decimalSeparator!)
+    let groupSeparator = Character(NumberFormatter().groupingSeparator!)
 
     var enteredAmountHandler: EnteredAmountHandler?
+    var numberOfDecimals: Int = 0
+    var defaultNumberOfDigits = 0
+    var formattedZeroAmount = "0"
 
-    private lazy var amountTextField: PasteOnlyTextField = {
-        let textField = PasteOnlyTextField(frame: .zero)
+    private lazy var amountTextField: AmountTextField = {
+        let textField = AmountTextField(frame: .zero)
         textField.textAlignment = .center
         textField.keyboardType = UIKeyboardType.decimalPad
         textField.delegate = self
         textField.accessibilityIdentifier = "transferAmountTextField"
         textField.setContentHuggingPriority(.defaultLow, for: .horizontal)
         textField.setContentCompressionResistancePriority(.defaultHigh, for: .horizontal)
         textField.addTarget(self, action: #selector(textFieldDidChange(_:)), for: .editingChanged)
+        textField.addTarget(self, action: #selector(didAmountTextFieldChange(_:)), for: .editingChanged)
```

Comment:
What's the difference between `didAmountTextFieldChange` and `textFieldDidChange`?  Can't the logic be added in the existing method

Reply:
Yes formatting logic can be added in textFieldDidChange(), I shall update that too. Thank You